### ISSUE TITLE
Can't claim my responsive design certificate.

### ISSUE THREAD

**Issue by RocktimSaikia** [freeCodeCamp/freeCodeCamp#17099]
I completed all the Responsive Design Projects successfully but now when i go to claim the certificate it tells to complete the projects first. What should i do.please help.I mean i already updated my projects and re-checked it and re-submitted it. But no luck.  It would be great if someone in freeCodeCamp community can help me with this issue.
As you can see here that i already completed the projects and it shoes in my profile too. But can not claim the Responsive certificate. Please fix it ASAP 
![complete](https://user-images.githubusercontent.com/33410545/39436848-d41b7c8c-4cbc-11e8-8588-1939e50adbc9.jpg)
![complete2](https://user-images.githubusercontent.com/33410545/39436849-d4947c2c-4cbc-11e8-95dc-ec9ade5a79f3.jpg)
Here is my profile link:
https://beta.freecodecamp.org/en/rocktimsaikia

ity help me out here.

**Comment by mstellaluna** [comment]
@RocktimSaikia hello, there are already open issues concerning not being able to claim certificates. Please refer to open issues https://github.com/freeCodeCamp/freeCodeCamp/issues/17031 and https://github.com/freeCodeCamp/freeCodeCamp/issues/16923

**Comment by RocktimSaikia** [comment]
@mstellaluna  yeah i saw those issues too. but  there is no solution.some of them were fixed by @QuincyLarson Himself. I wish if he could fix this  manually too.

**Comment by RocktimSaikia** [comment]
@raisedadead I see. So meaning , claiming certs wont be possible until  you ship beta .Is that what you meant ?

**Comment by raisedadead** [comment]
That is correct, please follow the open issues for progress.
